problems/prob1.rb

```diff
@@ -0,0 +1,10 @@
+#Given an integer array of size N, compute
+#the sum of all even numbers in this array
+
+def sum_of_even_numbers(numbers_array)
+  if numbers_array.nil? || numbers_array.empty? || numbers_array.any? { |e| e == nil  }
```

Comment:
Mathematically, sum of the elements of an empty set is 0, not nil. I think this is more naturally. Also, given the description of the problem, you won't need nil checks. The client code should receive an error if it doesn't use the function correctly